tests/suites/test_suite_ecdh.function

```diff
@@ -50,6 +50,8 @@ void ecdh_invalid_param( )
     mbedtls_ecp_keypair kp;
     int invalid_side = 42;
 
```

Comment:
There's a similar issue in `test_suite_ecjpake.function` in `ecjpake_invalid_param`.